internal/bgp/frr/config.go

```diff
@@ -351,10 +361,14 @@ func (s stringSet) Add(item string) {
 	s[item] = struct{}{}
 }
 
+// Elements returns the sorted slice of elements in the set.
 func (s stringSet) Elements() []string {
 	res := []string{}
 	for k := range s {
-		res = append(res, k)
+		i := sort.SearchStrings(res, k)
+		res = append(res, "")
+		copy(res[i+1:], res[i:])
+		res[i] = k
```

Comment:
Is this more effective than just sticking the keys in a slice and sorting it in the end? IMO the other way is a bit more readable.

Reply:
Good point, and I don't think it is. I just went straigth to "insert into sorted slice" and found a solution for that.
I don't think complexity wise it's different, and at this point we should not care about it if readability is improved.

Reply:
Done